src/System.Management.Automation/help/HelpCommands.cs

```diff
@@ -157,7 +157,12 @@ public SwitchParameter Examples
         /// Support WildCard strings as supported by WildcardPattern class.
         /// </remarks>
         [Parameter(ParameterSetName = "Parameters", Mandatory = true)]
-        public string Parameter { set; get; }
+        public string[] Parameter
+        {
+            get => _parameters;
+            set => _parameters = value;
+        }
+        private string[] _parameters;
```

Comment:
Why we use the private variable? Why not `public string[] Parameter { set; get; }`

Reply:
Agree.